### ISSUE TITLE
Load pharmpy when loading pharmr

### ISSUE THREAD

**Issue by stellabelin** [pharmpy/pharmr#118]
Instead of delaying until first call to Python code. We still need to be able to load pharmr without having Pharmpy installed since there is a function for installing Pharmpy.

**Comment by stellabelin** [comment]
Yes, so the previous system (see [here](https://github.com/pharmpy/pharmr/blob/main/R/zzz.R)) does this when importing Pharmpy, but this will make it unclear to the user when they load pharmr what is taking time, they probably will start by loading a model which will make it look like it's the model parsing that takes all the time, but the importing will take (admittedly little) time as well. The crux is that it needs to actually load pharmr since pharmr has a function for installing Pharmpy.

**Comment by stellabelin** [comment]
Maybe we can use a try-catch to see whether it can be loaded, but still allow pharmr to load (now that I'm rereading what you wrote maybe that's what you meant?)

**Comment by pharmpy-dev-123** [comment]
I guess that's what I meant, with a message to the user telling them that Pharmpy could not be loaded and they need to install it with command foobar.
